Symfony/CS/Tests/Fixer/Contrib/AlignConstantEqualsFixerTest.php

```diff
@@ -0,0 +1,243 @@
+<?php
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+class AlignConstantEqualsFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @param string $expected
+     * @param string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    /**
+     * @return array
+     */
+    public function provideFixCases()
+    {
+        return array(
+            array(
+                '<?php
+                const FIRST_CONSTANT          = 1;
+                const SECOND_CONSTANT         = 2;
+                const THIRD_AND_LAST_CONSTANT = 3;
+
+                $myVar = 1;
+                $mySecondVar = 1;
+                ',
+            ),
+            array(
+                '<?php
+                const FIRST_CONSTANT          = 1;
+                const SECOND_CONSTANT         = 2;
+                const THIRD_AND_LAST_CONSTANT = 3;
+                ',
+                '<?php
+                const FIRST_CONSTANT = 1;
+                const SECOND_CONSTANT = 2;
+                const THIRD_AND_LAST_CONSTANT = 3;
+                ',
+            ),
+            array(
+                '<?php
+                const FIRST_CONSTANT          = 1;
+                const SECOND_CONSTANT         = 2;
+                const THIRD_AND_LAST_CONSTANT = 3;
+                ',
+                '<?php
+                const FIRST_CONSTANT            = 1;
+                const SECOND_CONSTANT           = 2;
+                const THIRD_AND_LAST_CONSTANT   = 3;
+                ',
+            ),
+            array(
+                '<?php
+                const FIRST_CONSTANT          = 1;
+                const SECOND_CONSTANT         = 2;
+                const THIRD_AND_LAST_CONSTANT = 3;
+                ',
+                '<?php
+                const FIRST_CONSTANT          = 1;
+                const SECOND_CONSTANT         = 2;
+                const THIRD_AND_LAST_CONSTANT   = 3;
```

Comment:
Align fixer will not trim alignment to it's minimal form. It will take biggest alignment.
That is why your tests fail.
Yeah, it's sad, and you can change that behavior